pkg/chartvalues/apiextensions_release_e2e_template.go

```diff
@@ -1,9 +1,16 @@
 package chartvalues
 
-const apiExtensionsReleaseE2ETemplate = `namespace: {{ .Namespace }}
-operator:
-  name: {{ .Operator.Name }}
-  version: {{ .Operator.Version }}
+const apiExtensionsReleaseE2ETemplate = `active: {{ .Active }}
+authorities:
+  {{- range .Authorities }}
+  - name: "{{ .Name }}"
+    version: "{{ .Version }}"
+  {{- end }}
+date: {{ .Date }}
+name: {{ .Name }}
+namespace: {{ .Namespace }}
```

Comment:
I find this a bit strange because the namespace is not part of the spec. Should we maybe structure the values file like the CR itself? So we have a block for the object meta data and a block for the spec. That would be more intuitive. Food for thought. No blocker for now.